store/commitment/store.go

```diff
@@ -59,7 +67,7 @@ func (c *CommitStore) WriteBatch(cs *store.Changeset) error {
 	return nil
 }
 
-func (c *CommitStore) WorkingStoreInfos(version uint64) []store.StoreInfo {
+func (c *CommitStore) WorkingCommitInfo(version uint64) *store.CommitInfo {
```

Comment:
go doc

Reply:
added docs in `database.go/Committer Interface`

Reply:
thank you